src/sentry/tasks/weekly_reports.py

```diff
@@ -739,6 +741,8 @@ def render_template_context(ctx, user_id):
         "organizations:session-replay", ctx.organization
     ) and features.has("organizations:session-replay-weekly-email", ctx.organization)
 
+    notification_uuid = "".join(random.choices(string.ascii_letters + string.digits, k=16))
```

Comment:
@scttcper we should make this into some utility function cause we'll need it a lot

Reply:
jammed it somewhere